cost/azure/savings_plan/recommendations/CHANGELOG.md

```diff
@@ -1,5 +1,12 @@
 # Changelog
 
+## v1.3
+
+- Renamed `Savings Plan` policy_set from info block to `Savings Plans`.
```

Comment:
The feedback I provided for the other policy also applies to the CHANGELOG entries here.